File: 2026POTD/MARCH/05-03-2026/1758.java
class Solution {
    public int minOperations(String s) {
        StringBuilder sb=new StringBuilder(s);
        int steps=0;
        for(int i=1;i<sb.length();i++){
            if(sb.charAt(i-1)==sb.charAt(i) && sb.charAt(i)=='1'){
                sb.setCharAt(i,'0');
                steps++;
            }else if(sb.charAt(i-1)==sb.charAt(i) && sb.charAt(i)=='0'){
                sb.setCharAt(i,'1');
                steps++;
            }

        }
        return Math.min(steps,sb.length()-steps);
    }
}

File: 2026POTD/MARCH/05-03-2026/1758.cpp
class Solution {
public:
    int minOperations(string s) {
        int cnt = 0;
        for(int i = 0; i < s.size(); i++){
            if(s[i] != (i % 2 ? '1' : '0'))
                cnt++;
        }
        return min(cnt, (int)s.size() - cnt);
    }
};